expression/builtin_op_vec.go

```diff
@@ -190,11 +190,19 @@ func (b *builtinUnaryMinusRealSig) vecEvalReal(input *chunk.Chunk, result *chunk
 }
 
 func (b *builtinBitNegSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinBitNegSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalInt(b.ctx, input, result); err != nil {
+		return err
+	}
+	n := input.NumRows()
+	args := result.Int64s()
+	for i := 0; i < n; i++ {
+		args[i] = ^args[i]
```

Comment:
We should check ```input.IsNull(i)``` first.

Reply:
We can ignore this check to speed up since it doesn't affect the correctness of our results.